docs/kubectl-equivalence-in-java.md

```diff
@@ -0,0 +1,246 @@
+#Kubectl Equivalence in Java
+
+__TL;DR__: Get used to kubectl? Now our kubernetes Java client library has releases a set of 
+helpful client utilities which has the similar input argument interface as the kubectl binary.
+Especially the developers who're already familiar with kubectl commands, after reading this 
+document, you will know how to program to operate kubernetes as neat as kubectl.
```

Comment:
"you will know how to build programs that interact with Kubernetes as easily as `kubectl`"

Reply:
fixed